common/strings.h

```diff
@@ -96,9 +96,9 @@ class Strings {
 	static bool IsFloat(const std::string &s);
 	static const std::string ToLower(std::string s);
 	static const std::string ToUpper(std::string s);
-	static const std::string UcFirst(std::string s);
-	static std::string &LTrim(std::string &str, const std::string &chars = "\t\n\v\f\r ");
-	static std::string &RTrim(std::string &str, const std::string &chars = "\t\n\v\f\r ");
+	static const std::string UcFirst(const std::string& s);
+	static std::string &LTrim(std::string &str, std::string_view chars = "\t\n\v\f\r ");
+	static std::string &RTrim(std::string &str, std::string_view chars = "\t\n\v\f\r ");
```

Comment:
What's with the `string_view` change here

Reply:
std::string_view provides read-only access to an existing string (a C-style string literal, a std::string, or a char array) without need to initialize (or copy) a std::string.